config/environments/production.rb

```diff
@@ -56,6 +56,20 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.delivery_method = :smtp
+  host = 'bill-buck-generator.herokuapp.com/'
+  config.action_mailer.default_url_options = { host: host }
+
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    user_name: Rails.application.credentials.google[:email],
+    password: Rails.application.credentials.google[:password],
+    authentication: 'plain',
+    enable_starttls_auto: true
+  }
```

Comment:
I've never seen a mailer setup before. To send emails do we have to have an email address for the sender?

Reply:
This is the configuration for Rails to actually be able to send an email. I'm using a "service"
google account: bill.buck.generator@gmail.com